Lesson007/Task 3 HW07.Pyramid/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace Task3HW07.Pyramid
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int[,] arrey = new int[4, 5];
+
+            for (int y = 0; y < arrey.GetLength(0); y++) //высота массива
+            {
+                for (int x = 0; x < arrey.GetLength(1); x++) // ширина массива
+                {
+                    arrey[1, 2] = 1;
+                    arrey[2, 1] = 1;
+                    arrey[2, 2] = 1;
+                    arrey[2, 3] = 1;
+                    arrey[3, 0] = 1;
+                    arrey[3, 1] = 1;
+                    arrey[3, 2] = 1;
+                    arrey[3, 3] = 1;
+                    arrey[3, 4] = 1;
+
+                    Console.Write(arrey[y, x] + "\t");
+                }
```

Comment:
need to rewrite thit logic. its equal that we do this =>
Console.Write("H");
Console.Write("e");
Console.Write("l");
Console.Write("l");
Console.Write("o");
Console.Write(" ");
Console.Write("w");
Console.Write("o");
Console.Write("r");
Console.Write("l");
Console.Write("d");

Reply:
Fixed))